src/settings/DisplaySettingsForm.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'react-final-form';
+
+import {
+  Button,
+  Checkbox,
+  Layout,
+  Pane,
+} from '@folio/stripes/components';
+
+import stripesFinalForm from '@folio/stripes/final-form';
+
+const DisplaySettingsForm = ({
+  handleSubmit,
+  label,
+  pristine,
+  submitting,
+}) => (
+  <form id="courses-display-settings-form" onSubmit={handleSubmit}>
+    <Pane
+      defaultWidth="fill"
+      fluidContentWidth
```

Comment:
Oh, I've never heard of this — and it's not documented at https://github.com/folio-org/stripes-components/blob/master/lib/Pane/readme.md

I hope it will fix the horrible rendering problems in single-record-import settings when the window is resized. I'll try that. Nice side-effect of reviewing!

Reply:
... no, sadly this isn't the solution to that. The single-record-import settings page uses `<ControlledVocab>`, and the `<Pane>` that this creates already has the undocumented `fluidContentWidth` property. Oh well.